### ISSUE TITLE
Scaffold with decimal data type 

### ISSUE THREAD

**Issue by jazarja** [redwoodjs/redwood#1969]
Redwoodjs generate graphql data type when we're using Decimal in prisma schema.

Step to reproduce:
Prisma schema 

```
model Service {
  id    Int     @id @default(autoincrement())

  title  String
  description  String?
  price Decimal

  state Int?

  expiredBy DateTime?
  publishedBy DateTime?

  createdAt DateTime @default(now())
}
```

Output graphql sdl:

```  
type Service {
    id: Int!
    title: String!
    description: String
    price: Decimal!
    state: Int
    expiredBy: DateTime
    publishedBy: DateTime
    createdAt: DateTime!
  }
```

Expected error
```
Error: Unknown type "Decimal"
```

**Comment by dthyresson** [comment]
Hi @jazarja . 

Just to clarify, are you reporting that `Decimal` causes and error when scaffolding (or generating SDL) -- or that you *expect* it to raise and error, and it is not.

From my understanding, Prisma does support Decimal:

https://www.prisma.io/docs/reference/api-reference/prisma-schema-reference#decimal

```js
model User {
  email    String  @unique
  number   Decimal   @default(22.99)
}
```

Also, in which version of RedwoodJS, that database (Postgres, SQLite, etc) and Prisma was this observed?

Thanks!

**Comment by dthyresson** [comment]
In a new RW app (version 27) and SQLite:

```js
model UserExample {
  id    Int     @id @default(autoincrement())
  email String  @unique
  name  String?
  cost  Decimal
}
```

then `yarn rw g scaffold UserExample`

There is a warning about Decimal:

![image](https://user-images.githubusercontent.com/1051633/110955170-4d578580-8317-11eb-895e-3d3f0aabefa7.png)

The used prisma studio to add a record:

![image](https://user-images.githubusercontent.com/1051633/110955505-ade6c280-8317-11eb-8ecf-ad7c35e61590.png)

but on launch on `yarn rw dev`:

`api | Error: Unknown type "Decimal".

So, the issue here seems to be that Decimal is not supported.

**Comment by dthyresson** [comment]
It seems that using Postgres with native types a workaround it to use `Float` in the SDL.

```js
model UserExample {
  id    Int     @id @default(autoincrement())
  email String  @unique
  name  String?
  cost  Decimal @DS.Decimal(35, 8)
}
```

and then in the SDL:

```js
  type UserExample {
    id: Int!
    email: String!
    name: String
    cost: Float!
  }
```

Note that also worked without native types and a schema of simply:

```js
model UserExample {
  id    Int     @id @default(autoincrement())
  email String  @unique
  name  String?
  cost  Decimal
}
```

Should the SDL for Decimal when scaffolding be Float?

I do not see a [graphql-scalars](https://github.com/Urigo/graphql-scalars#readme) type for `Decimal`.

**Comment by cannikin** [comment]
Related: https://github.com/redwoodjs/redwood/issues/2859

**Comment by thedavidprice** [comment]
This seems resolved by #2860, correct?

**Comment by cannikin** [comment]
Yup, I used "Closes #1969" in the commit message so it should have automatically closed this one...not sure why it didn't? Maybe because I listed two issues...each needs to be on its own line?
